### DISCUSSION TITLE
incomprehensible characters

### DISCUSSION THREAD

**Discussion by kaizoku75** [SWHL/RapidVideOCR#39] [Q&A]
problem incomprehensible characters in srt file
https://pastebin.com/uXvd48pT

**Comment by SWHL** [comment]
Could you please provide the original video file?
I will reproduce and debug the effect.

**Comment by kaizoku75** [comment]
https://suzihaza.com/f/zn7-3bj4ng-0e62

**Comment by SWHL** [comment]
- The RapidVideOCR only supports extraction of Chinese and English subtitles.
- By looking at the original video file, I found that the subtitles of the file are not one of Chinese and English languages.

**Comment by RoyRiv3r** [comment]
>The RapidVideOCR only supports extraction of Chinese and English subtitles.

Hi, Is that possible to support latin languages?

**Comment by SWHL** [comment]
> The RapidVideOCR only supports extraction of Chinese and English subtitles.
> 
> Hi, Is that possible to support latin languages?

The models in this [link](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/models_list_en.md#23-multilingual-recognition-modelupdating) are all supported.

**Reply by Mamacito473** [reply]
How do we import a french model in the rapidvideoOCR Directory , I dont get it ?

https://i.imgur.com/0xJUr5r.png

**Reply by SWHL** [reply]
Please wait a moment, I will give a full documentation.
